src/routes/aboutUs.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+
+const router = express.Router();
+const aboutUs = require("../api/Controller/aboutUs");
+// api/contact/
+router
+  .route("/")
+  /* api/contact : to create collection of any enquiry from contact us page */
```

Comment:
I think comment is reversed , post API should be retired later or we need some sms-dev-admin-write access .